easybuild/easyblocks/n/neuron.py

```diff
@@ -60,85 +62,133 @@ def extra_options():
         extra_vars = {
             'paranrn': [True, "Enable support for distributed simulations.", CUSTOM],
         }
-        return ConfigureMake.extra_options(extra_vars)
+        extra_vars.update(ConfigureMake.extra_options())
+        extra_vars.update(CMakeMake.extra_options())
+        return extra_vars
 
     def configure_step(self):
         """Custom configuration procedure for NEURON."""
+        if LooseVersion(self.version) < LooseVersion('7.8'):
```

Comment:
this should be 

```suggestion
        if LooseVersion(self.version) < LooseVersion('7.8.1'):
```
? 
(7.8 was still autotools)